notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/BasicNotificationMessage.java

```diff
@@ -0,0 +1,68 @@
+package com.microsoft.windowsazure.messaging.notificationhubs;
+
+import androidx.annotation.NonNull;
+
+import java.util.Map;
+
+/**
+ * A basic implementation of NotificationMessage.
+ */
+class BasicNotificationMessage implements NotificationMessage {
+
+    /**
+     * Notification title. Will be null when Notification is received from click in system tray.
+     */
+    private final String mTitle;
+
+    /**
+     * Notification message. Will be null when Notification is received from click in system tray.
+     */
+    private final String mMessage;
+
+    /**
+     * Key/value pairs sent with a Push Notification.
+     */
+    private final Map<String, String> mCustomData;

```

Comment:
Just to be consistent, we should name this `mData`.

Reply:
Fixed in c0beaae34fce3e8c21d6a61ee80f0ee43147433e